spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 require 'rspec'
 require 'pathname'
 require 'webmock/rspec'
+require 'gooddata'
```

Comment:
So that it is automatically required in specs. It helps reduce duplication and is a common practice. Travis does it too: https://github.com/travis-ci/travis-api/blob/master/spec/spec_helper.rb#L15